NDB.Covid19/NDB.Covid19/Locales/en.json

```diff
@@ -111,7 +111,7 @@
   "WELCOME_PAGE_WHATS_NEW_BULLET_ONE": "Smittestopp now works in EU and allows you to receive a message if you encounter infected EU citizens with a COVID-19 app in Norway. ",
   "WELCOME_PAGE_WHATS_NEW_BULLET_TWO": "Smittestopp now helps to stop the spread of COVID-19 in EU countries. You can give your consent to share your status so European COVID-19 app users can receive a message if you have been nearby – no one will get to know who you are or where you are from.",
   "WELCOME_PAGE_WHATS_NEW_BUTTON": "Next",
-  "WELCOME_PAGE_WHATS_NEW_FOOTER": "Because of the new features you have to give your consent again on the next page.",
+  "WELCOME_PAGE_WHATS_NEW_FOOTER": "If you would like to withdraw your consent because of these changes, you can do this in the \"Consent and privacy\" page in the main menu.",
```

Comment:
Instead of "main menu" please call it "Menu"

Reply:
done